Tests/PostgresNIOTests/New/Extensions/PSQLFrontendMessage+Equatable.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 14.01.21.
+//
+
+import class Foundation.JSONEncoder
```

Comment:
Ditch Xcode template headers.
```suggestion
import class Foundation.JSONEncoder
```